torchrl/modules/models/models.py

```diff
@@ -463,6 +463,21 @@ def _make_net(self, device: Optional[DEVICE_TYPING]) -> nn.Module:
             layers.append(Squeeze2dLayer())
         return layers
 
+    def forward(self, inputs: torch.Tensor) -> torch.Tensor:
+
+        if len(inputs.shape) == 5:
```

Comment:
Here's how I would approach this:
```python
*batch, C, L, W = input.shape
if len(batch) > 1:
    inputs = inputs.flatten(0, len(batch))
out = super(ConvNet, self).forward(inputs)
if len(batch) > 1:
    out = out.unflatten(0, batch)
```

Reply:
Makes sense this is more general. Just changed it.

Only changed this line, apparently otherwise also colapses the len(batch) dimension which is the number of channels.
`inputs = inputs.flatten(0, len(batch) - 1)
`